presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -626,15 +653,49 @@ void PrestoServer::yieldTasks() {
   }
 }
 
+#ifdef __linux__
+class BatchThreadFactory : public folly::NamedThreadFactory {
+ public:
+  explicit BatchThreadFactory(const std::string& name)
+      : NamedThreadFactory{name} {}
+
+  std::thread newThread(folly::Func&& func) override {
+    return folly::NamedThreadFactory::newThread(
+        [func_2 = std::move(func)]() mutable {
+          sched_param param;
+          param.sched_priority = 0;
```

Comment:
Have we tuned on this? Thanks!

Reply:
This is the only supported value by `SCHED_BATCH`. It fails when I set it to any different value.